### ISSUE TITLE
Multiple blank task bars load

### ISSUE THREAD

**Issue by Magnus-Deus** [ad2017gd/RainbowTaskbar#58]
Upon starting, a metric ton of buggy taskbars load, clogging up system resources.

**Screenshots**
![Screenshot 2024-07-29 140915](https://github.com/user-attachments/assets/76374514-a042-437f-982e-559ff5f5b098)

**Windows version and/or build**
![Screenshot 2024-07-29 141323](https://github.com/user-attachments/assets/d700bfba-d0f7-42c6-8155-813a13bedc12)

**RainbowTaskbar version**
Version 2.4.1

**Comment by ad2017gd** [comment]
Well that's new. Any other taskbar or shell modifications?

**Comment by Magnus-Deus** [comment]
As far as I remember, no.

**Comment by Magnus-Deus** [comment]
Update: I have got it to work with just my laptop alone, but with my two other monitors it does the duplicating thing.

**Comment by aashish108** [comment]
Yeah I think I get the same issue, loads try to load for some reason then it crashes.

**Comment by ad2017gd** [comment]
So it seems both of you have an issue on a triple monitor setup. Fair enough since I never got to test it with more than 2.

**Comment by ad2017gd** [comment]
you guys think you could be available to try an experimental build to see if it fixes the issue? at the moment i have no way of testing a 3 monitor setup

**Comment by aashish108** [comment]
> you guys think you could be available to try an experimental build to see if it fixes the issue? at the moment i have no way of testing a 3 monitor setup

Yeah sure. Let me know.

**Comment by aashish108** [comment]
> you guys think you could be available to try an experimental build to see if it fixes the issue? at the moment i have no way of testing a 3 monitor setup

Did you mean for me to try v3.0b?

**Comment by ad2017gd** [comment]
no i have yet to make a special build to test this problem. ill send it here when i have time

**Comment by ad2017gd** [comment]
honestly i have no idea what the issue could be from. i ve tried using normal windows as taskbars and i dont seem to have this issue when i have 3 windows or more. theres also kinda no way for me to test with more than 2 monitors

**Comment by MamiyaOtaru** [comment]
triple monitor setup here (win11).  Nothing shows up on screen, but the rnbtsk-x64.exe process uses a lot of memory and the system gets laggy until I kill it.  Doesn't happen with only one monitor.  

Can right click on the tray icon and get the menu, but clicking the menu just makes the menu disappear without the action happening (exit, or open editor)

**Comment by ad2017gd** [comment]
can someone confirm if this issue still persists on latest update?
